index.js

```diff
@@ -33,7 +33,8 @@ module.exports = {
 				'unicorn/new-for-builtins': 'error',
 				'unicorn/regex-shorthand': 'error',
 				'unicorn/prefer-spread': 'error',
-				'unicorn/error-message': 'error'
+				'unicorn/error-message': 'error',
+				'unicorn/unsafe-regex': 'error'
```

Comment:
`unsafe-regex` sounds like we enforce unsafe regexes. I think it should be `no-unsafe-regex`.